src/components/Header.tsx

```diff
@@ -0,0 +1,290 @@
+import { MenuOutlined } from "@ant-design/icons";
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useTranslation } from "react-i18next"; 
+import { handleNavigate } from "../utils/navigation.ts"; 
+import { i18n as I18nInstance } from "i18next";
+
+interface AdBanner {
+  id: number;
+  image: string;
+}
+
+const changeLanguage = (
+  i18n: I18nInstance,
+  lng: string,
+  setActiveLang: React.Dispatch<React.SetStateAction<string>>
+) => {
+  i18n.changeLanguage(lng);
+  setActiveLang(lng);
+};
+
+//Array
+const menuItems = [
+  {
+    title: "film",
+    children: [
+      { title: "now_showing", action: "/now-showing" },
+      { title: "coming_soon", action: "/coming-soon" },
+    ],
+  },
+  {
+    title: "cinemas",
+    children: [
+      { title: "all_cinemas", action: "/cinemas" },
+      { title: "special_cinemas", action: "#" },
+      { title: "3d_cinemas", action: "#" },
+    ],
+  },
+  {
+    title: "members",
+    children: [
+      { title: "cgv_account", action: "/account-info" },
+      { title: "benefits", action: "#" },
+    ],
+  },
+  {
+    title: "cultureplex",
+    children: [
+      { title: "online_store", action: "#" },
+      { title: "rent_and_group", action: "#" },
+      { title: "e_cgv", action: "#" },
+      { title: "egift", action: "#" },
+      { title: "cgv_rules", action: "#" },
+    ],
+  },
+];
+
+export const Header: React.FC = () => {
+  const navigate = useNavigate();
+  const { t, i18n } = useTranslation(); 
+  const [activeLang, setActiveLang] = useState(i18n.language);
+  const [isNavbarOpen, setIsNavbarOpen] = useState(false);
+  const [activeMenu, setActiveMenu] = useState<string | null>(null);
+  const [adBanner, setAdBanner] = useState<AdBanner | null>(null);
+  const accessToken = false; // Mocked
+  const user = { hoTen: "User" }; // Mocked
+
+  // Fetch API
+  useEffect(() => {
+    const fetchAdBanner = async () => {
+      try {
+        const response = await fetch("http://localhost:5000/adBanner");
```

Comment:
```suggestion
        const response = await fetch(`${APP_API_HOST}/adBanner`);
```
Đưa http://localhost:5000 vào file môi trường .env e nha